qcodes/dataset/sqlite/queries.py

```diff
@@ -1594,7 +1594,9 @@ def get_metadata(conn: ConnectionPlus, tag: str, table_name: str) -> str:
                             "result_table_name", table_name)
 
 
-def get_metadata_from_run_id(conn: ConnectionPlus, run_id: int) -> Dict:
+def get_metadata_from_run_id(
+        conn: ConnectionPlus, run_id: int
+) -> Dict[Any, Any]:
```

Comment:
i guess it's a bit too tedious to make it Dict[str, Any] because sqlite can potentially return not only strings?